src/CSV_Parser.php

```diff
@@ -359,7 +361,7 @@ public function parse($headers = true) {
 	 * @throws Jabran\Exception\InvalidEncodingException
 	 * @return self
 	 */
-	public function encode() {
+	private function encode() {
```

Comment:
What is the reason for this change?

Reply:
Reading your issue, I had understood that the encode method was not to be accessed otherwise than by the parse method. I'm sorry if I misunderstood.

Reply:
Originally I kept it public so to be covered directly via unit tests and can be used for any reason if needed by a consumer. But I see that it can be a privately scoped and then tested via `parse()` method. Would you like to make all those changes or create a new issue of this and revert this scope change?

Reply:
Are we going ahead anymore on this?